modules/common.nix

```diff
@@ -3,119 +3,127 @@
 # SPDX-License-Identifier: MPL-2.0
 
 { lib, pkgs, config, options, inputs, ... }:
-
 {
   imports = [
     ./services/nginx.nix
     ./ssh-hostkeys.nix
     ./nix-gc.nix
   ];
 
-  networking.firewall = {
-    allowPing = false;
-    logRefusedConnections = false;
+  options.services.nginx.addHSTSHeaders = lib.mkOption {
```

Comment:
There are headers other than HSTS, maybe we should name this option more accurately

Reply:
Fair point